dashboard/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+app_name = "dashboard"
+urlpatterns = [
+    path("admissionstaff/", views.admissionstaff, name="admissionstaff"),
```

Comment:
> This could've been implemented using a single path as done for the login and register apps and handle the redirection to the specific pages from the view instead of from url. That was we have consistency across the application in regards to redirection based on userType.

solved